### PR TITLE
Updated badges from the README

### PR DESCRIPTION
**Pull request by vladciobancai**
Updated badges picture with the helm chart version tested

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Resources used by the Starter Kit include the following:
...
-    [![loki-stack](https://img.shields.io/static/v1?label=loki-stack&message=2.4.1&color=success)](https://artifacthub.io/packages/helm/grafana/loki-stack/2.4.1)
-    [![velro](https://img.shields.io/static/v1?label=velero&message=2.23.6&color=success)](https://artifacthub.io/packages/helm/vmware-tanzu/velero/2.23.6)
-    ![triliovault](https://img.shields.io/static/v1?label=triliovault&message=2.6.3&color=success)
-    [![sealed-secrets](https://img.shields.io/static/v1?label=sealed-secrets&message=1.16.1&color=success)](https://artifacthub.io/packages/helm/bitnami-labs/sealed-secrets/1.16.1)
+    [![ingress-nginx](https://img.shields.io/static/v1?label=ingress-nginx&message=4.0.13&color=success)](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.0.13)
+    [![ingress-ambassador](https://img.shields.io/static/v1?label=ingress-ambassador&message=6.9.3&color=success)](https://artifacthub.io/packages/helm/datawire/ambassador/6.9.3)
+    [![prometheus-stac](https://img.shields.io/static/v1?label=prometheus-stack&message=30.0.0&color=success)](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack/30.0.0)
```
Comment (v-ctiutiu) [inline README.md:19]: Please update `kube-prometheus-stack` badge to `30.0.1` as mentioned in #103 review. Thanks.

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vladciobancai) [reply README.md]: Updated the version

File: README.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Resources used by the Starter Kit include the following:
...
-    [![loki-stack](https://img.shields.io/static/v1?label=loki-stack&message=2.4.1&color=success)](https://artifacthub.io/packages/helm/grafana/loki-stack/2.4.1)
-    [![velro](https://img.shields.io/static/v1?label=velero&message=2.23.6&color=success)](https://artifacthub.io/packages/helm/vmware-tanzu/velero/2.23.6)
-    ![triliovault](https://img.shields.io/static/v1?label=triliovault&message=2.6.3&color=success)
-    [![sealed-secrets](https://img.shields.io/static/v1?label=sealed-secrets&message=1.16.1&color=success)](https://artifacthub.io/packages/helm/bitnami-labs/sealed-secrets/1.16.1)
+    [![ingress-nginx](https://img.shields.io/static/v1?label=ingress-nginx&message=4.0.13&color=success)](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.0.13)
+    [![ingress-ambassador](https://img.shields.io/static/v1?label=ingress-ambassador&message=6.9.3&color=success)](https://artifacthub.io/packages/helm/datawire/ambassador/6.9.3)
+    [![prometheus-stac](https://img.shields.io/static/v1?label=prometheus-stack&message=30.0.1&color=success)](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack/30.0.1)
```
Comment (v-ctiutiu) [inline README.md:19]: Seems to be a typo here:`prometheus-stac` should be `prometheus-stack`.

File: README.md [hunk 3]
```diff
@@ -14,13 +14,13 @@ Resources used by the Starter Kit include the following:
...
+    [![loki-stack](https://img.shields.io/static/v1?label=loki-stack&message=2.5.1&color=success)](https://artifacthub.io/packages/helm/grafana/loki-stack/2.5.1)
+    [![velro](https://img.shields.io/static/v1?label=velero&message=2.23.6&color=success)](https://artifacthub.io/packages/helm/vmware-tanzu/velero/2.27.3)
+    ![triliovault](https://img.shields.io/static/v1?label=triliovault&message=2.27.3&color=success)
```
Comment (v-ctiutiu) [inline README.md:22]: Triliovault version should be `2.6.3`.

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vladciobancai) [reply README.md]: Updated the name to be `prometheus-stack.`

File: README.md [hunk 4]
```diff
@@ -14,13 +14,13 @@ Resources used by the Starter Kit include the following:
...
-    ![triliovault](https://img.shields.io/static/v1?label=triliovault&message=2.6.3&color=success)
-    [![sealed-secrets](https://img.shields.io/static/v1?label=sealed-secrets&message=1.16.1&color=success)](https://artifacthub.io/packages/helm/bitnami-labs/sealed-secrets/1.16.1)
+    [![ingress-nginx](https://img.shields.io/static/v1?label=ingress-nginx&message=4.0.13&color=success)](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.0.13)
+    [![ingress-ambassador](https://img.shields.io/static/v1?label=ingress-ambassador&message=6.9.3&color=success)](https://artifacthub.io/packages/helm/datawire/ambassador/6.9.3)
+    [![prometheus-stac](https://img.shields.io/static/v1?label=prometheus-stack&message=30.0.1&color=success)](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack/30.0.1)
+    [![loki-stack](https://img.shields.io/static/v1?label=loki-stack&message=2.5.1&color=success)](https://artifacthub.io/packages/helm/grafana/loki-stack/2.5.1)
+    [![velro](https://img.shields.io/static/v1?label=velero&message=2.23.6&color=success)](https://artifacthub.io/packages/helm/vmware-tanzu/velero/2.27.3)
```
[the remark this answers is not in the document]
Comment (vladciobancai) [reply README.md]: The typo has been fixed and the link updated to the correct version

File: README.md (hunk 3 above)
Comment (vladciobancai) [reply README.md]: Updated the version to the correct one

**Review by v-ctiutiu — APPROVED** [review]
Looks good to me now.

**Pull request merged**